### DISCUSSION TITLE
更新1.7.8版本后，代理配置-绕过-接口选项没了

### DISCUSSION THREAD

**Discussion by rsgdn** [nikkinikki-org/OpenWrt-nikki#162] [General]
![屏幕截图 2024-09-03 203442](https://github.com/user-attachments/assets/f14e5f8c-f1ec-4f5b-98bb-84616e4c33d2)
![屏幕截图 2024-09-03 203644](https://github.com/user-attachments/assets/2502e3e8-2ef2-473a-ad12-ec1965589d79)

**Comment by morytyann** [comment]
嗯，因为不需要了，不用再设置了
